### DISCUSSION TITLE
Backward compatibility between CPIX versions ( v2.0, v2.2 and v2.3)

### DISCUSSION THREAD

**Discussion by myjara13** [Dash-Industry-Forum/DASH-IF-IOP#412] [Q&A] [answered]
Hi all,

I would like to confirm  if there is **backward compatibility** between CPIX versions, or put in another way, if the DRM is supporting CPIX 2.3, does it also mean DRM can support CPIX 2.2 and CPIX 2.0?

Many thanks

**Comment by lpiron** [comment, marked as answer]
The short answer is yes.
CPIX carries key information and DRM information. As long as the DRM can provide DRM information in a format that is supported by CPIX (PSSH, HLS signaling DASH ContentProtection ...), it is compatible. This support is there in all CPIX 2.x versions.
Hope this helps.

**Comment by myjara13** [comment]
HI Ipiron,

Thank you so much for your feedback, it's very useful. Thanks
